src/router/index.js

```diff
@@ -0,0 +1,33 @@
+import Vue from "vue";
+import VueRouter from "vue-router";
+import Home from "../views/Home.vue";
+import NotFound from "../views/NotFound.vue";
+import Details from "../views/Details";
```

Comment:
Consider using the consistent path for the entire application.
Webpack configuration in Vue allows the usage of "@" as an alias for root of the application.